hypothesis-python/src/hypothesis/internal/conjecture/dfa/__init__.py

```diff
@@ -195,6 +203,44 @@ def pop():
                 pop()
         return cache[i]
 
+    @cached
+    def has_strings(self, state, length):
+        """Returns the number of strings of length ``length``
+        that are accepted when starting from state ``state``."""
```

Comment:
Looks like this docstring was left over from a copy-paste.

Reply:
Oops. It was.